src/components/ebay-carousel/examples/10-autoplay/template.marko

```diff
@@ -0,0 +1,19 @@
+<style>
+    .demo7-card {
```

Comment:
The demo class setup isn't great here to begin with, but you're probably redeclaring the `demo7-...` classes from above.

Reply:
Good catch thanks!